adal/build.gradle

```diff
@@ -22,6 +22,7 @@ android {
 
     defaultConfig {
         minSdkVersion 14
+        //noinspection OldTargetApi
         targetSdkVersion 23
```

Comment:
// comments the line. delete instead of commenting. i see you suppressed it in the xml

---
In reply to: [73977550](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/694#discussion_r73977550) [](ancestors = 73977550)